workers/fetcher.go

```diff
@@ -19,27 +18,51 @@ const (
 )
 
 //Type - Type of the api query
-type Type string
+type FetchType string
 
 const (
 	//Member - member type
-	Member Type = "members"
+	Members FetchType = "members"
 )
 
 //Fetch - Fetches the data from the propublica api
-func Fetch(congress int, chamber Chamber, ty Type, end string, response interface{}) error {
-	query := fmt.Sprintf("%s%d/%s/%s%s", base, congress, chamber, ty, end)
+func PropublicaMembersFetch(
+	congress int,
+	chamber Chamber,
+	response interface{},
+) error {
+	url := fmt.Sprintf(
+		"%s%d/%s/%s",
+		endpoints.GetEndpoint(endpoints.Propublica),
+		congress,
+		chamber,
+		"members.json",
+	)
+
+	return fetch(url, response)
+}
+
+func PropublicaMemberFetch(
+	memberUrl string,
+	response interface{},
+) error {
+
+	return fetch(memberUrl, response)
+}
+
+func fetch(url string, response interface{}) error {
```

Comment:
I was confused how to handle the other types of api calls, Now i'm clear this approach is nice and clean 👍